### PR TITLE
AEROGEAR-8100 - App identifier no more

### PR DESCRIPTION
**Pull request by aliok**
https://issues.jboss.org/browse/AEROGEAR-8100

* Removed app identifier in backend and in frontend

### CODE REVIEW

File: ui/src/components/overview/MobileClientCardView.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ class MobileClientCardView extends Component {
       <EmptyState>
         <EmptyStateTitle>{this.emptyStateMessage.noAppsCreated}</EmptyStateTitle>
         <EmptyStateAction>
-          <CreateClient createButtonSize="large" item={new MobileApp().toJSON()} />
+          <CreateClient createButtonSize="large" />
```
Comment (aliok) [inline ui/src/components/overview/MobileClientCardView.js]: addressing https://github.com/aerogear/mobile-developer-console/pull/120#discussion_r233778903

**Comment by psturc** [comment]
There's an error when I try to bind the service
screen shot 2018-11-16 at 09 54 49

**Comment by aliok** [comment]
@secondsun can you have a look at that problem please Summers? I didn't understand why we need to submit the form in the first place. Anyway, `this.form` there is a React component I think and it doesn't have a `submit` method.

Thanks for the help

**Comment by secondsun** [comment]
@aliok  update your dependencies.  Mozilla's form library had an update.  You need to call submit to trigger the validation logic.

**Comment by aliok** [comment]
ok, gotta update deps

**Comment by aliok** [comment]
@psturc @wei-lee @ziccardi @wojta  ready for another review.

**Comment by wei-lee** [comment]
👀

**Pull request merged**